### ISSUE TITLE
Local .babelrc breaks scss-tokenizer

### ISSUE THREAD

**Issue by nybblr** [sasstools/scss-tokenizer#10]
I just clean installed node-sass (same patch version I think) and through a dependency of a dependency it now includes scss-tokenizer. I'm getting this cryptic error:

```
$ ./node_modules/.bin/node-sass

/Users/jonathan/my-app/node_modules/scss-tokenizer/node_modules/babel-core/lib/transformation/file/options/option-manager.js:126
      if (!option) this.log.error("Unknown option: " + alias + "." + key, ReferenceError);
                           ^

TypeError: Cannot read property 'error' of undefined
    at OptionManager.mergeOptions (/Users/jonathan/my-app/node_modules/scss-tokenizer/node_modules/babel-core/lib/transformation/file/options/option-manager.js:126:28)
    at OptionManager.addConfig (/Users/jonathan/my-app/node_modules/scss-tokenizer/node_modules/babel-core/lib/transformation/file/options/option-manager.js:107:10)
    at OptionManager.findConfigs (/Users/jonathan/my-app/node_modules/scss-tokenizer/node_modules/babel-core/lib/transformation/file/options/option-manager.js:168:35)
    at OptionManager.init (/Users/jonathan/my-app/node_modules/scss-tokenizer/node_modules/babel-core/lib/transformation/file/options/option-manager.js:229:12)
    at compile (/Users/jonathan/my-app/node_modules/scss-tokenizer/node_modules/babel-core/lib/api/register/node.js:117:22)
    at normalLoader (/Users/jonathan/my-app/node_modules/scss-tokenizer/node_modules/babel-core/lib/api/register/node.js:199:14)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/jonathan/my-app/node_modules/scss-tokenizer/node_modules/babel-core/lib/api/register/node.js:216:7)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
```

I tried a few things, then nuked my local `.babelrc` config and `node-sass` worked. So my guess is `scss-tokenizer` is using my local .babelrc automatically?

Perhaps adding an explicit config to this package would prevent that?

**Comment by nybblr** [comment]
Ahh, this might also do the trick: https://babeljs.io/docs/usage/babelrc/#lookup-behavior

I'll test adding this option locally and see if it works.

**Comment by nybblr** [comment]
Ah, since this is using an older version of Babel, adding this to scss-tokenizer's `package.json` prevents the lookup behavior:

```json
"babel": {
  "breakConfig": true
}
```

Quick PR, or is there another PR in progress that will fix this issue?

**Comment by fozzle** [comment]
Having the same issue.

**Comment by xzyfer** [comment]
Thanks all. I unpublished sass-graph 2.2.0 but for some reason people were still getting it. Looks like there may be an issue for yarn users - https://github.com/yarnpkg/yarn/issues/3274

Testing #11 locally before publishing a new release.

**Comment by xzyfer** [comment]
- released scss-tokenizer@0.2.1 with addition fixes
- released sass-graph@2.2.2

Thanks all!
